packages/test/test-end-to-end-tests/src/test/localTestSignals.spec.ts

```diff
@@ -133,20 +133,20 @@ describeFullCompat("TestSignals", (getTestObjectProvider) => {
             }
         });
 
-        user1ContainerRuntime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
+        user1ContainerRuntime.containerRuntime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
             if (message.type === "TestSignal") {
                 user1HostSignalReceivedCount += 1;
             }
         });
 
-        user2ContainerRuntime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
+        user2ContainerRuntime.containerRuntime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
             if (message.type === "TestSignal") {
                 user2HostSignalReceivedCount += 1;
             }
         });
 
         user1ContainerRuntime.submitSignal("TestSignal", true);
-        await waitForSignal(user1ContainerRuntime, user2ContainerRuntime);
+        await waitForSignal(user1ContainerRuntime.containerRuntime, user2ContainerRuntime.containerRuntime);
```

Comment:
This test is failing for the same reason as above. `ContainerRuntime` will only emit a "signal" event for signals sent from `ContainerRuntime`, not for signals sent from `DataStoreRuntime`/`DataStoreContext`.